### PR TITLE
Add public build for exlude AVC/HEVC codec

### PR DESCRIPTION
**Pull request by stellawuintel**

### CODE REVIEW

**Review by Sherry-Lin — APPROVED** [review]
looks good

File: .github/workflows/ubuntu.yml [hunk 1]
```diff
@@ -752,3 +752,81 @@ jobs:
...
+        cmake -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_INSTALL_LIBDIR=/usr/lib/x86_64-linux-gnu ..
+        make VERBOSE=1 -j$(nproc)
+        sudo make install
+    - name: build media-driver
+      run: |
+        cd media
+        mkdir build && cd build
+        cmake -DENABLE_NONFREE_KERNELS=OFF \
+          -DENABLE_KERNELS:BOOL=OFF \
+          -DBUILD_KERNELS:BOOL=OFF \
```
Comment (XinfengZhang) [inline .github/workflows/ubuntu.yml:816]: it means, it only enable the HW feature without media kernel support
even vp8 DP is excluded?

Comment (stellawuintel) [reply .github/workflows/ubuntu.yml]: Updated to more feature build

**Review by XinfengZhang — APPROVED** [review]

**Comment by stellawuintel** [comment]
#1522

**Pull request merged**